.travis.yml

```diff
@@ -34,6 +35,7 @@ before_install:
   - uname -r
   - sudo apt-get -q update
   - sudo apt-get install -y libseccomp-dev/trusty-backports
+  - docker run --rm -v /usr/local/bin:/target jpetazzo/nsenter
```

Comment:
why?

Reply:
This is required by the `ocicni` package and also a functionality called `portforward` in CRI.

Basically what we need is to run some code inside a target network namespace, which can't be done easily in golang today. https://github.com/containerd/cri-containerd/issues/144

Reply:
We need golang 1.10 to get rid of nsenter https://github.com/golang/go/issues/20676.

Reply:
WHAT!!!!!?????  How does an image fix this?

Reply:
Done.